src/pages/Compare/index.spec.tsx

```diff
@@ -886,6 +916,287 @@ describe(__filename, () => {
     expect(dispatch).not.toHaveBeenCalled();
   });
 
+  it('dispatches fetchDiff and fetchVersionFile for the next file with diff', () => {
+    const addonId = 10;
+    const baseVersionId = 44;
+    const headVersionId = 55;
+    const file1 = 'file1.json';
+    const file2 = 'file2.json';
+    const version = createExternalVersionWithEntries(
+      [
+        { path: file1, status: 'M' },
+        { path: 'fileWithoutChange', status: '' },
+        { path: file2, status: 'M' },
+      ],
+      { id: headVersionId },
+      // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Any time you need this, I suggest putting it at the top of the file like `/* eslint-disable ... */`. It's a silly rule and we should just remove it altogether.